src/components/Modal/ReturningUserModal.js

```diff
@@ -33,6 +33,7 @@ const StyledForm = styled('form')({
 
   '@media (max-width: 768px)': {
     width: '75vw',
+    height: '35vh',
```

Comment:
@alderwhiteford not sure why this was here when I pulled main, is this something you added? It's not currently in main soooo

Reply:
Are you adding the height back in?

Reply:
Idk lol I didn’t add that, it was there after I merged main. I’m going to assume it’s a conflict that slipped in and get rid of it